src/otx/v2/adapters/torch/mmengine/modules/hooks/adaptive_training_hook.py

```diff
@@ -87,15 +100,16 @@ def before_train_iter(self, runner: Runner) -> None:
                     hook.interval = adaptive_interval
             self._initialized = True
 
-    def get_adaptive_interval(self, iter_per_epoch: int) -> int:
+    def get_adaptive_interval(self, iter_per_epoch) -> int:
```

Comment:
Nitpick: Why removed `int` type annotation?

Reply:
Revert changes in daec976